api.md

```diff
@@ -431,12 +434,72 @@ PostGIS geospatial indexes).  During testing, you might check your
 total chunk sizes via the [`chunk_relation_size`](#chunk_relation_size)
 function.
 
-**Space partitions**: In most cases, it is advised for users not to use
+**Space partitions:** In most cases, it is advised for users not to use
 space partitions. The rare cases in which space partitions may be useful
 are described in the [add dimension][] section.
 
 ---
 
+## create_distributed_hypertable() [](create_distributed_hypertable)
+
+A specialized implementation of [`create_hypertable`](#create_hypertable)
+designed for creating hypertables distributed across multinode 
+environments.  The required and optional arguments are the same as for
+`create_hypertable`, as is the returned value.  The main difference is
+that this command will default the `replication_factor` optional
+parameter to a value of 1.
+
+#### Sample Usage [](create_distributed_hypertable-examples)
+
+Create a table `conditions` spread across the entire distributed
+database.
+```sql
+SELECT create_distributed_hypertable('conditions', 'time');
+```
+
+Create a table `conditions` which will be partitioned across data
+nodes by the 'location' column.  Note that the number of space
+partitions is automatically equal to the number of data nodes in this
+case.
+```sql
+SELECT create_distributed_hypertable('conditions', 'time', 'location');
+```
+
+Create a table `conditions` using a specific set of data nodes.  Each
+chunk will be written to two data nodes.
+```sql
+SELECT create_distributed_hypertable('conditions', 'time', 'location', replication_factor => 2,
+    data_nodes => '{ "data_node_1", "data_node_2", "data_node_4", "data_node_7" }');
+```
+
+#### Best Practices [](create_distributed_hypertable-best-practices)
+
+**Space partitions:** As opposed to the normal [`create_hypertable` best practices]
+(#create_hypertable-best-practices), space partitions are highly
+recommended for distributed hypertables.  Incoming data will be
+divided among data nodes based upon the space partition (the first one
+if multiple space partitions have been defined).  However, if there is
+no space partition, all the data for a time slice will be written to a
+single data node, negating much of the benefit of distributing the
```

Comment:
There might be other upsides to avoiding space partitioning, so I think we should be careful about giving too much of on opinion here without having tested pros and cons of different setups.  I am happy to see us recommend space partitioning, but we probably shouldn't say much about other setups until we know more.

For instance, we could recommend smaller chunk time intervals in case people choose to not use space partitioning? With small chunks, you would quite quickly be able to use all nodes and you would also get better parallelism for queries that e.g., aggregate a single device (like TSBS single-groupby-1).

Reply:
Removed the "negating much of the benefit..." section.